packages/expo/cli/api/graphql/generated.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
Wouldn't that just be overwritten next time?

Reply:
That's a good point. Perhaps a .eslintignore file with `cli/api/graphql/generated.ts` would be simple and work well.

Reply:
This file doesn't have a generation step it's pulled from eas-cli and acts as a placeholder until we have a more formal API package for the Expo API. Can add the ignore file but it doesn't change the results.